src/molecules/Footer/footer.module.css

```diff
@@ -0,0 +1,12 @@
+footer {
+  padding: 20px 28px;
+  background-color: var(--footer-background-color);
+  display: flex;
+  justify-content: space-between;
+}
+
+footer a {
+  font-weight: bold;
+  text-decoration: none;
+  color: white;
+}
```

Comment:
Please arrange properties in alphabetical order.

Reply:
Done.